src/messages/FileMessage.vue

```diff
@@ -25,12 +64,39 @@ export default {
     messageColors: {
       type: Object,
       required: true
+    },
+  },
+  methods:{
+    humanReadableFileSize(aSize) {
+      const prettyBytes = require('pretty-bytes');
```

Comment:
This should be `import`ed above like other modules. And can be passed to `methods` to be used in the template directly:

```js
methods: {
  prettyBytes,
  smartTruncate(str, len) {
    return smartTruncate(str, len, {position: 10})
  },
}
```

No semi-colons. There's also inconsistent white-space. Please run `npm run lint` and update the diff.